src/components/ui/Card.vue

```diff
@@ -1,23 +1,59 @@
 <template>
-  <div class="card">
+  <div class="card" v-on:click="onCardClicked">
     <slot></slot>
   </div>
 </template>
 
 <script>
 export default {
   name: 'Card',
+  methods: {
+    /**
+     * Determine if card either has a 'primary link' defined by '.card__link'
+     * or if the clicked item is itself an anchor tag. If it has either, redirect
+     * according to link requirements.
+     */
+    onCardClicked(e) {
```

Comment:
I really get this. We're trying to support different / multiple links in the cards. Moving this to JS breaks things like _long press on → open in new tab_ on mobile or _middle click to open in new tab_ on desktop.
I would vote for either talking to @mbransn about using one the earlier proposals (not having the card clickable or not having a link in the text and make the whole card a link) or accept the original pseudo element version.

What do you think?